SF iOS/SF iOS.xcodeproj/project.pbxproj

```diff
@@ -1162,17 +1217,20 @@
 			baseConfigurationReference = 657137982D0CEF4F9D8C04C9 /* Pods-Coffup.release.xcconfig */;
 			buildSettings = {
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
+				CLANG_ENABLE_MODULES = YES;
 				CODE_SIGN_ENTITLEMENTS = Coffup.entitlements;
 				"CODE_SIGN_IDENTITY[sdk=iphoneos*]" = "iPhone Developer";
 				CODE_SIGN_STYLE = Automatic;
 				CURRENT_PROJECT_VERSION = 17;
 				DEVELOPMENT_TEAM = F4WTN36U2F;
 				INFOPLIST_FILE = "SF iOS/Info.plist";
-				IPHONEOS_DEPLOYMENT_TARGET = 11.0;
+				IPHONEOS_DEPLOYMENT_TARGET = 13.0;
```

Comment:
We could either do this or ifdef everything.

Reply:
How would this effect current users not on iOS 13?

Reply:
I believe it would not be available to them in the App Store